DOC-2148: AI Assistant bug fix documentation entry for TINY-10102 in the 6.6.1 Release Notes (#2831)
tinymce/tinymce-docs: changelog.md

The format is loosely based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).

### Unreleased

> - DOC-2148: documentation of bug fix, _when Safari on macOS was the host environment and a returned response was inserted into the {productname} editor, the inserted content did not scroll into view_, added to **AI Assistant** section of `6.6.1-release-notes.adoc`.

Author: ```suggestion
- DOC-2148: documentation of bug fix, _When a response from the AI Assistant dialog was inserted into a selection not in view, the content did not scroll into view_, added to **AI Assistant** section of `6.6.1-release-notes.adoc`.
```
